src/solvers/flattening/boolbv_get.cpp

```diff
@@ -38,14 +36,14 @@ exprt boolbvt::get(const exprt &expr) const
       std::vector<bool> unknown;
       bvt bv;
       std::size_t width=map_entry.width;
-
+      DATA_INVARIANT(
+        map_entry.width <= map_entry.literal_map.size(),
+        "I'm not sure what exactly these mean");
```

Comment:
As mentioned here, I don't actually know what an appropriate description would be. This is why this commit is marked as WIP